README.md

```diff
@@ -341,15 +341,15 @@ EXAMPLES
   $ superface create:profile starwars/character-information --station
 ```
 
-_See code: [dist/commands/create/profile.ts](https://github.com/superfaceai/cli/tree/main/src/commands/create/profile.ts)_
+_See code: [dist/commands/create/profile.js](https://github.com/superfaceai/cli/blob/v3.0.1/dist/commands/create/profile.js)_
```

Comment:
issue: but we don't commit `dist` folder to git? How could this link possibly work?

Reply:
This is autogenerated by oclif and I missed the wrong path :|